misk-admin/src/main/kotlin/misk/web/dashboard/AdminDashboardModule.kt

```diff
@@ -20,6 +20,7 @@ class AdminDashboardModule(private val isDevelopment: Boolean) : KAbstractModule
 
   override fun configure() {
     install(BaseDashboardModule(isDevelopment))
+    install(ConfigDashboardTabModule(isDevelopment))
```

Comment:
We can now safely reinstall this since it is using the default SECURE mode and will not show any insecure YAML config, only the JVM and other future secure config.